convlab/base_models/llm/base.py

```diff
@@ -23,6 +24,8 @@ def __init__(self, api_type, model_name_or_path, system_instruction=None, genera
                 self.model = ChatGLM2(model_name_or_path)
             else:
                 self.model = HFModels(model_name_or_path)
+        elif api_type == 'litellm':
```

Comment:
why create a separate provider for this ? 

Why not just have all the LLMs go through LiteLLM

Reply:
Hi @ishaan-jaff , because `HFModels` and `ChatGLM2` classes are for local hf models instead of API calling, and `openai` class is the most widely used one so I would like to expose its own interface for simplification. For more LLMs through API, users can use litellm.